src/com/hms_networks/americas/sc/fileutils/FileAccessManager.java

```diff
@@ -0,0 +1,74 @@
+package com.hms_networks.americas.sc.fileutils;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+
+/**
+ * Class for accessing (reading and writing) files on the Ewon Flexy filesystem.
```

Comment:
Maybe rephrase this comment. Since file access is done in a standard way if you were just accessing the file you wouldn't need this. They are more of file access helpers/utils.

Reply:
changed